packages/kbn-sandbox/src/native/sandbox_linux.cpp

```diff
@@ -0,0 +1,68 @@
+#include "sandbox.hpp"
+
+#include <linux/seccomp.h>
+#include <linux/filter.h>
+#include <sys/syscall.h>
+#include <stddef.h>
+#include <errno.h>
+#include <sys/prctl.h>
+
+#include <string.h> /* for strerror */
+
+// execveat was added in Linux 3.19
+#ifndef __NR_execveat
+#define JMP_START 4
+#else
+#define JMP_START 5
+#endif
+
+struct sock_filter reject_syscalls[] = {
+  // TODO(sissel): Verify arch of syscall
```

Comment:
It's this one line that Elasticsearch has: https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/bootstrap/SystemCallFilter.java#L373

I dont' think this TODO is a blocker because the code as-is provides good protection. Adding an architecture check is a good idea for added safety, but the execve blocking is the main objective here especially as Kibana doesn't ship with node-ffi (right?). I recommend making this a future task to deliver, but not block this PR.

Reply:
Kibana doesn't ship with node-ffi. When making the changes to always compile in the `execveat` syscall filter, I went ahead and made the changes to check the architecture and check the _X32_SYSCALL_BIT similar to how Elasticsearch enforces it with the limit. Mind double-checking the changes I made @jordansissel ?